mdpow/workflows/dihedrals.py

```diff
@@ -181,11 +181,16 @@ def rdkit_conversion(u, resname):
 
     """
 
+    from .base import guess_elements
+    # temporary
+    # keep here to avoid circular imports?
+
```

Comment:
I currently don't see that base imports dihedrals in any way, or did I overlook something? What's your rationale for not putting the imports at the top of the file?